Cargo.toml

```diff
@@ -65,13 +65,13 @@ lazy_static = "1.4.0"
 uuid-0_8 = { package = "uuid", version = "0.8.1", features = ["serde", "v4"], optional = true }
 uuid = { version = "1.1.2", features = ["serde", "v4"] }
 serde_bytes = "0.11.5"
-serde_with = { version = "1", optional = true }
+serde_with1 = { package = "serde_with", version = "1.3.1", optional = true }
```

Comment:
I think we need to keep the existing one without renaming it so that existing users turning on the `serde_with` feature will continue to get the same version.